src/server/req_stat.c

```diff
@@ -722,6 +758,11 @@ req_stat_svr(struct batch_request *preq)
 	update_license_ct(&server.sv_attr[(int)SRV_ATR_license_count],
 		server.sv_license_ct_buf);
 
+	if (preq->rq_fromsvr) {
```

Comment:
fromsvr or do you want to check if it is coming from the scheduler?

Reply:
Is there a better way to check that it's coming from the scheduler? I noticed that this flag is set when it is requested by sched so I used it.

Reply:
I think you check the connection against the scheduler connection.  Look for any place we log a connection coming from the scheduler and see how it does it.  If it uses this flag, then great.

Reply:
Ok ya, I think the other parts look at the job, get its psched object and compare against the sched connection. Since this is server stat we'll have to loop over the list of all sched objects and match the connection. Should I do that instead?

Reply:
you can do this:
```c
conn_t *conn = get_conn(preq->rq_conn);
if ((conn->cn_authen & PBS_NET_CONN_TO_SCHED) != 0)) {
   < connection to sched >
} else {
   < connection not to sched >
}
```
or

```c
if (strcmp(preq->rq_user, PBS_SCHED_DAEMON_NAME) == 0) {
  < connection to sched >
} else {
  < connection not to sched >
}
```

Reply:
For example you can see process_request() when its change rq_user to PBS_SCHED_DAEMON_NAME if connection is to sched.

Reply:
Thanks a lot Hiren, I've used your first suggestion since it seems to be faster than comparing strings, and it seems to work.

Reply:
I would test this with a multisched just incase there is a difference between the default and a non-default scheduler.

Reply:
I have a PTL test which checks that multisched works fine, and it passed after these changes. see "test_multisched_multival"